assets/xml/overlays/ovl_En_Tru.xml

```diff
@@ -0,0 +1,24 @@
+<Root>
+    <File Name="ovl_En_Tru" BaseAddress="0x80A85620" RangeStart="0x33B0" RangeEnd="0x5C30" Segment="128">
```

Comment:
```suggestion
    <File Name="ovl_En_Tru" BaseAddress="0x80A85620" RangeStart="0x33B0" RangeEnd="0x5C30">
```
I could be wrong but based on the other xmls, segment is unnecessary here and actually causes a warning with ZAPD.